lib/govspeak.rb

```diff
@@ -406,6 +406,20 @@ def self.devolved_options
       %(<iframe class="govspeak-embed-video" width="500" height="281" src="#{embed_url}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>)
     end
 
+    extension("Figure", /#{NEW_PARAGRAPH_LOOKBEHIND}\$Figure\s*$(.*?)\s*\$EndFigure/m) do |figure|
+      alt = figure.scan(/\$Alt\s*(.*?)\s*\$EndAlt/m)[0][0]
+      url = figure.scan(/\$URL\s*(.*?)\s*\$EndURL/m)[0][0]
+      caption = figure.scan(/\$Caption\s*(.*?)\s*\$EndCaption/m)[0][0]
+      %(<figure class="image embedded">
+         <div class="img">
+          <img src="#{url}" alt="#{alt}">
```

Comment:
Do you need a closing tag here?

Reply:
I took the govspeak readme image output as a guide and that doesn't have a closing tag. Not sure why, it better to add one in?

Reply:
I read up on it a bit more, and https://stackoverflow.com/a/23890795 claims we shouldn't have a closing tag (because image tags cannot have children, so closing tag serves no purpose). 

Which explains why other govspeak version doesn't use it :)